projects/crc-timing/testing/test_crc_timing.py

```
@@ -0,0 +1,302 @@
+# not using `os.getcwd()` anymore because of
+# https://stackoverflow.com/questions/1542803/is-there-a-version-of-os-getcwd-that-doesnt-dereference-symlinks
+RUN_DIR = pathlib.Path(os.getenv('PWD')) # for run_one_matbench
+os.chdir(TOPSAIL_DIR)
+
+import prepare_crc_timing
+
+def start_macos_vm(base_work_dir):
+    _, _, vfkit_binary_path = prepare_crc_timing.get_crc_binary_path(base_work_dir)
+
+    try:
+        old_vfkit_pid = remote_access.read("/tmp/vfkit.pid").strip()
```

Ruff (0.13.1)

Probable insecure usage of temporary file or directory: "/tmp/vfkit.pid"

(S108)